src/components/Drawer/DrawerHeader.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import type { ComponentProps, FC } from 'react';
+import { useId } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { MdClose, MdHome } from 'react-icons/md';
+import { mergeDeep } from '../../helpers/merge-deep';
+import type { DeepPartial } from '../../types';
+import { useDrawerContext } from './DrawerContext';
+import type { FlowbiteBoolean } from '../Flowbite';
+
+export interface FlowbiteDrawerHeaderTheme {
+  inner: {
+    titleIcon: string;
+    titleText: string;
+    closeButton: string;
+    closeIcon: string;
+  };
+  collapsed: FlowbiteBoolean;
+}
+
+export interface DrawerHeaderProps extends ComponentProps<'div'>, Record<string, unknown> {
+  theme?: DeepPartial<FlowbiteDrawerHeaderTheme>;
+  titleIcon?: FC<ComponentProps<'svg'>>;
+  closeIcon?: FC<ComponentProps<'svg'>>;
+  title?: string;
+}
+
+export const DrawerHeader: FC<DrawerHeaderProps> = ({
+  children,
+  className,
+  theme: customTheme = {},
+  closeIcon: CloseIcon,
+  titleIcon: TitleIcon,
+  title,
+  ...props
+}) => {
+  const id = useId();
+
+  const { isOpen, onClose, theme: rootTheme } = useDrawerContext();
+
+  const theme = mergeDeep(rootTheme.header, customTheme);
+
+  return (
+    <div className={twMerge(className)} {...props}>
+      <h5 className={twMerge(theme.inner.titleText)}>
+        {TitleIcon ? (
+          <TitleIcon aria-hidden className={twMerge(theme.inner.titleIcon)} />
+        ) : (
+          <MdHome aria-hidden className={twMerge(theme.inner.titleIcon)} />
+        )}
+        {title}
+      </h5>
+
+      <button
+        type="button"
+        className={twMerge(theme.inner.closeButton)}
+        title="Close"
+        aria-label="Close Drawer"
```

Comment:
I think it's important for the users to be able to override the aria-label for this button here specifically

Reply:
would it be possible to give a code example? as if we use `<Drawer.Header />` component this way, then how we can use users `arial-label`? suggestions please 🙏